contracts/validators/CredentialAtomicQueryMTPValidator.sol

```diff
@@ -79,4 +91,14 @@ contract CredentialAtomicQueryMTPValidator is CredentialAtomicQueryValidator {
 
         return params;
     }
+
+    function _getSpecialInputPairs(
+        bool hasSelectiveDisclosure
+    ) internal pure override returns (ICircuitValidator.KeyInputIndexPair[] memory) {
+        ICircuitValidator.KeyInputIndexPair[]
+            memory pairs = new ICircuitValidator.KeyInputIndexPair[](2);
+        pairs[0] = ICircuitValidator.KeyInputIndexPair({key: "userID", inputIndex: 1});
+        pairs[1] = ICircuitValidator.KeyInputIndexPair({key: "timestamp", inputIndex: 10});
+        return pairs;
+    }
```

Comment:
could be moved to CredentialAtomicQueryValidator since userID and timestamp have same indexes for sig and mtp